src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -775,24 +775,20 @@ private static String processPageSourceForGodMode(String content) {
                 .replace("\"/_ah", "\"${web.url}/_ah")
                 .replaceAll("V[0-9]\\.[0-9]+", "V\\${version}")
                 // photo from instructor
-                .replaceAll(Const.ActionURIs.STUDENT_PROFILE_PICTURE + "\\?" + Const.ParamsNames.STUDENT_EMAIL + "=([a-zA-Z0-9]){1,}\\&"
+                .replaceAll(Const.ActionURIs.STUDENT_PROFILE_PICTURE + "\\?" + Const.ParamsNames.STUDENT_EMAIL + "=([a-zA-Z0-9]){1,}\\&amp;"
                         + Const.ParamsNames.COURSE_ID + "=([a-zA-Z0-9]){1,}", 
                         Const.ActionURIs.STUDENT_PROFILE_PICTURE + "\\?" + Const.ParamsNames.STUDENT_EMAIL 
-                        + "={*}\\&" + Const.ParamsNames.COURSE_ID + "={*}")
-                .replaceAll(Const.ActionURIs.STUDENT_PROFILE_PICTURE + "\\?" + Const.ParamsNames.COURSE_ID + "=([a-zA-Z0-9]){1,}\\&"
+                        + "={*}\\&amp;" + Const.ParamsNames.COURSE_ID + "={*}")
+                .replaceAll(Const.ActionURIs.STUDENT_PROFILE_PICTURE + "\\?" + Const.ParamsNames.COURSE_ID + "=([a-zA-Z0-9]){1,}\\&amp;"
                         + Const.ParamsNames.STUDENT_EMAIL + "=([a-zA-Z0-9]){1,}", 
                         Const.ActionURIs.STUDENT_PROFILE_PICTURE + "\\?" + Const.ParamsNames.COURSE_ID 
-                        + "={*}\\&" + Const.ParamsNames.STUDENT_EMAIL + "={*}")
+                        + "={*}\\&amp;" + Const.ParamsNames.STUDENT_EMAIL + "={*}")
```

Comment:
All the above are undoing what #4336 did. Let me explain why: #4336 missed the actual cause of the failure.
I've done a little research and found out that _just before_ the regeneration, `HtmlHelper` _still_ has `Sanitizer` involved. This fact alone would've suggested that the regex match fails _not_ because of the `&amp;` being un-sanitized back to `&`. And it really is not.
Then what fails it? It is `Const.ActionURIs.STUDENT_PROFILE_PICTURE` _being sanitized_ in `HtmlHelper`. God mode only targets _unsanitized_ `Const.ActionURIs.STUDENT_PROFILE_PICTURE`.
Then how come all these while they always pass? Surprise, surprise, #4361 is the answer!